wp/lib/bap_wp/src/precondition.mli

```diff
@@ -177,6 +188,9 @@ val num_unroll : int ref
     - an architecture of x86_64
     - freshening variables set to false
     - and using all function input registers when generating function symbols at a call site
+    - the default range of addresses of the stack: [0x00007fffffff0000, 0x00007fffffffffff]
```

Comment:
Maybe we should describe also what these inputs are supposed to do rather than just their default values. It seems tedious though so I won't insist too much.